src/main/java/com/dynxsty/dih4jda/InteractionHandler.java

```diff
@@ -111,7 +111,15 @@ protected InteractionHandler(@Nonnull DIH4JDA dih4jda) {
 				findSlashCommands(pkg);
 				findContextCommands(pkg);
 			} catch (ReflectiveOperationException | DIH4JDAException e) {
-				DIH4JDALogger.error("An error occurred while initializing commands in package %s: %s", pkg, e.getMessage());
+				if (e.getCause() instanceof IllegalArgumentException) {
+					DIH4JDALogger.warn("Failed to load requirements in package %s: %s", pkg, e.getCause().getMessage());
+					continue;
+				}
+				String message = e.getMessage();
+				if (message == null) {
+					message = e.getCause().getMessage();
+				}
```

Comment:
And what the flip is this?? Just do 
```suggestion
				String message = e.getMessage() == null ? e.getCause().getMessage() : e.getMessage();
```